### PR TITLE
addings fixes for debug

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

* If you have developed tools active, it will output this new warning in the top nav bar
* We have a new development key to use the Query Monitor plugin output log to show data that are pulled from the external API using PHP.
* All development checks now also check if the top level Dashboard key is active, so you can turn off the main feature and everything else will turn off

Screenshot 2023-10-25 at 15 36 27
Screenshot 2023-10-25 at 15 36 37
Screenshot 2023-10-25 at 15 37 55

### CODE REVIEW

**Review by kancijan — APPROVED** [review]

File: src/AdminMenus/FormAdminBarMenu.php [hunk 1]
```diff
@@ -80,17 +80,23 @@ public function getTopBarMenu(WP_Admin_Bar $adminBar): void
...
 				'id' => $prefix,
 				'parent' => null,
-				'group'  => null,
-				'title' => \esc_html__('Eightshift Forms', 'eightshift-forms'),
-				'href'  => Helper::getListingPageUrl(),
+				'group' => null,
+				'title' => ($isDevelopMode || $isDevelopModeQmLog) ? $mainLabel . Helper::getProjectIcons('warning') : $mainLabel,
+				'href' => Helper::getListingPageUrl(),
+				'meta' => [
+					'title' => ($isDevelopMode || $isDevelopModeQmLog) ? \esc_html__('Your Debug tools are active!', 'eightshift-forms') : $mainLabel,
```
Comment (goranalkovic-infinum) [inline src/AdminMenus/FormAdminBarMenu.php:98]:
```suggestion
					'title' => ($isDevelopMode || $isDevelopModeQmLog) ? \esc_html__('Debug tools are active!', 'eightshift-forms') : $mainLabel,
```

File: src/Blocks/components/admin-top-menu-bar/admin-top-menu-bar-admin.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#wp-admin-bar-es-forms > a > svg { // stylelint-disable-line selector-max-specificity
+	color: red;
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/admin-top-menu-bar/admin-top-menu-bar-admin.scss:2]:
```suggestion
	color: #FF6900;
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Review by piqusy — CHANGES_REQUESTED** [review]
Lovely improvement 👏🏼 

Left two suggestions for your consideration

**Review by dadadavorin — APPROVED** [review]

File: src/Blocks/components/admin-top-menu-bar/admin-top-menu-bar-admin.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (iruzevic) [reply src/Blocks/components/admin-top-menu-bar/admin-top-menu-bar-admin.scss:2]: it actually is really important as you can leak sensitive information to an unauthorized users

**Pull request merged**